Keen.NET.Test/QueryTest.cs

```diff
@@ -1138,12 +1138,12 @@ public void Serialize_SimpleValue_Success()
         {
             var filter = new QueryFilter("prop", QueryFilter.FilterOperator.Equals(), "val");
 
-            var json = JObject.FromObject(filter).ToString();
+            var json = JObject.FromObject(filter).ToString(Newtonsoft.Json.Formatting.None);
```

Comment:
Thanks, we really need to stop embedding strings like this for these tests, probably as part of fixing Issue #118.